src/directives/jf_data_list/components/jf_data_list_item/jf_data_list_item.js

```diff
@@ -0,0 +1,93 @@
+export default function jfDataListItem() {
+
+	return {
+		restrict: 'E',
+		scope: {
+			item: '=',
+			index: '<'
+		},
+		templateUrl: 'directives/jf_data_list/components/jf_data_list_item/jf_data_list_item.html',
+		require: '^jfDataList',
+		controller: jfDataListItemController,
+		bindToController: true,
+		controllerAs: '$ctrl'
+	}
+}
+
+class jfDataListItemController {
+	constructor($scope, JFrogModal, $rootScope, $element, JFrogUIUtils, $compile) {
+		this.$scope = $scope;
+		this.JFrogModal = JFrogModal;
+		this.$rootScope = $rootScope;
+		this.$element = $element;
+		this.JFrogUIUtils = JFrogUIUtils;
+		this.$compile = $compile;
+	}
+
+	/**
+	 * htmlIsOverflowing(rowId)
+	 * Get al the row's children.
+	 * Then sum all the children's box model width (includes padding and margin) in a loop
+	 * If the child already has the overflowing class => remove it
+	 * When the sum gets > then the container's width => add the overflowing class to him
+	 * After exiting the loop return the overflowing flag
+	 * **/
+	htmlIsOverflowing(rowId) {
+		let elem = this.$element.find(rowId);
```

Comment:
Quick fix suggestion 
```suggestion
		let elem = this.$element.find(rowId);
		if(elem.length) {
		    let children = elem.children('.tag');
		let maxWidth = elem.closest('.data-list-item-value').outerWidth() - 60;
		let totalChildrenWidth = 0;
		children.each((i,child) => {
			let childElem = $(child);
			totalChildrenWidth += childElem.outerWidth()
				+ parseInt(childElem.css('margin-left'))
				+ parseInt(childElem.css('margin-right'));
 			if(totalChildrenWidth < maxWidth){
				childElem.removeClass('overflowing-child');
			}
			if(totalChildrenWidth > maxWidth && !childElem.is('.overflowing-child')){
				childElem.addClass('overflowing-child');
			}
 		});
		return elem.children('.tag.overflowing-child').length > 0;
		}
		return false;
```